vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -204,6 +204,9 @@ def process_host_hardware_disks(extra_attributes)
           # Logical index contains number of logical disks
           # TODO(lsmola) For now ignoring smart data, that are in format e.g. sda{cciss,1}, we need to design
           # how to represent RAID
+          disk_size = normalize_blank_property_num(disks.fetch_path(disk, 'size'))
+          # Convert the size from GB to B
+          disk_size = disk_size ? (disk_size * 1_024**3) : 0
```

Comment:
yes, I'll  make it part of the normalize_blank_property_num

property.try(:to_i) || 0, should solve all cases I think